src/components/custom-field-input-multiple-choice/custom-field-input-multiple-choice.jsx

```diff
@@ -152,6 +158,9 @@ function CustomFieldInputMultipleChoice(props) {
                 fill="skip"
                 name={iconClear.id}
                 onClick={onChoicesClear}
+                onEnter={clearChoices}
+                titleAttrText={'Remove selected choice'}
```

Comment:
Oh, did we find an issue with `ariaLabel`? We should choose one. On that point, I think it makes sense to keep the `on ${props.label}` since we want to indicate the context for the button.